fineract-progressive-loan-embeddable-schedule-generator/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/EmbeddableProgressiveLoanScheduleGenerator.java

```diff
@@ -30,10 +31,11 @@ public class EmbeddableProgressiveLoanScheduleGenerator {
     private final ScheduledDateGenerator scheduledDateGenerator;
     private final EMICalculator emiCalculator;
 
-    public EmbeddableProgressiveLoanScheduleGenerator() {
+    public EmbeddableProgressiveLoanScheduleGenerator(LoanTransactionProcessingService loanTransactionProcessingService) {
```

Comment:
We dont need incoming parameter of LoanTransactionProcessingService, handle inside as null instead.